unit_tests/test_server.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+import unittest
+sys.path.append(os.path.join(os.getcwd(), '..'))
+from common.variables import RESPONSE, ERROR, USER, ACCOUNT_NAME, TIME, ACTION, PRESENCE

```

Comment:
There're no the common.variables module and other modules.

Reply:
I added files